ext/standard/mt_rand.c

```diff
@@ -303,16 +303,17 @@ PHPAPI zend_long php_mt_rand_common(zend_long min, zend_long max)
 /* {{{ Returns a random number from Mersenne Twister */
 PHP_FUNCTION(mt_rand)
 {
-	zend_long min;
-	zend_long max;
+	zend_long min = 0;
+	zend_long max = ZEND_LONG_MAX;
 	int argc = ZEND_NUM_ARGS();
 
 	if (argc == 0) {
 		// genrand_int31 in mt19937ar.c performs a right shift
 		RETURN_LONG(php_mt_rand() >> 1);
 	}
```

Comment:
These blocks should come after ZPP and you need to make the function work with one argument now for this to work.